test/Table_Tests/src/Common_Table_Operations/Main.enso

```diff
@@ -88,6 +88,8 @@ type Test_Selection
        - supports_unicode_normalization: Specifies if the backend compares
          strings taking Unicode Normalization into account, i.e. whether
          's\u0301' is considered equal to 'ś'.
+       - is_collation_en_gb_utf8: Specifies if the backend is running with the
```

Comment:
Not a great name, but I'm not sure of the right way to handle this; I cannot reproduce the difference locally.

Reply:
It feels like this is actually redundant with the `order_by_unicode_normalization_by_default` flag. Please check if we can avoid adding this flag at all.

Reply:
I think there are two different settings here -- `order_by_unicode_normalization_by_default` is whether it can use normalization in ordering, and collation is about what the ordering is. I wasn't able to try different collations locally; the setting didn't seem to matter. I think this needs further exploration, but not for this test fix.

Reply:
No, I don't agree.

That is exactly the same thing. This switches if the ordering uses normalization or not. I think there is only one valid ordering (for this example) if Unicode normalization is used. At least that's what I meant in this test by "order_by_unicode_normalization_by_default".

So the two options really are redundant for the purposes of these tests, there does not seem to be any point in keeping them separate.

Also note that this setting is only used in this 1 test.